src/Model/Notify.php

```diff
@@ -0,0 +1,775 @@
+						'seen'  => $notify['seen']
+					];
+					break;
+
+				case Activity::ATTENDMAYBE:
+					$formattedNotify = [
+						'label' => 'attendmaybe',
+						'link'  => $this->baseUrl->get(true) . '/display/' . $notify['parent-guid'],
+						'image' => ProxyUtils::proxifyUrl($notify['author-avatar'], false, ProxyUtils::SIZE_MICRO),
+						'url'   => $notify['author-link'],
+						'text'  => $this->l10n->t("%s may attend %s's event", $notify['author-name'], $notify['parent-author-name']),
+						'when'  => $default_item_when,
+						'ago'   => $default_item_ago,
+						'seen'  => $notify['seen']
+					];
+					break;
+
+				case Activity::FRIEND:
+					if (!isset($notify['object'])) {
+						$formattedNotify = [
+							'label' => 'friend',
+							'link'  => $default_item_link,
+							'image' => $default_item_image,
+							'url'   => $default_item_url,
+							'text'  => $default_item_text,
+							'when'  => $default_item_when,
+							'ago'   => $default_item_ago,
+							'seen'  => $notify['seen']
+						];
+						break;
+					}
+					/// @todo Check if this part here is used at all
+					$this->logger->info('Complete data.', ['notify' => $notify, 'callStack' => System::callstack(20)]);
+
+					$xmlHead         = "<" . "?xml version='1.0' encoding='UTF-8' ?" . ">";
+					$obj             = XML::parseString($xmlHead . $notify['object']);
+					$notify['fname'] = $obj->title;
+
+					$formattedNotify = [
+						'label' => 'friend',
+						'link'  => $this->baseUrl->get(true) . '/display/' . $notify['parent-guid'],
+						'image' => ProxyUtils::proxifyUrl($notify['author-avatar'], false, ProxyUtils::SIZE_MICRO),
+						'url'   => $notify['author-link'],
+						'text'  => $this->l10n->t("%s is now friends with %s", $notify['author-name'], $notify['fname']),
+						'when'  => $default_item_when,
+						'ago'   => $default_item_ago,
+						'seen'  => $notify['seen']
+					];
+					break;
+
+				default:
+					$formattedNotify = [
+						'label' => $default_item_label,
+						'link'  => $default_item_link,
+						'image' => $default_item_image,
+						'url'   => $default_item_url,
+						'text'  => $default_item_text,
+						'when'  => $default_item_when,
+						'ago'   => $default_item_ago,
+						'seen'  => $notify['seen']
+					];
+			}
+
+			$formattedNotifies[] = $formattedNotify;
+		}
+
+		return $formattedNotifies;
+	}
+
+	/**
+	 * Get network notifications
+	 *
+	 * @param bool $seen    False => only include notifications into the query
+	 *                            which aren't marked as "seen"
+	 * @param int        $start   Start the query at this point
+	 * @param int        $limit   Maximum number of query results
+	 *
+	 * @return array [string, array]
+	 *    string 'ident' => Notification identifier
+	 *    array 'notifications' => Network notifications
+	 *
+	 * @throws Exception
+	 */
+	public function getNetworkNotifies(bool $seen = false, int $start = 0, int $limit = self::DEFAULT_PAGE_LIMIT)
```

Comment:
`Notify` already is the name of the class, this should be renamed `getNetworkList`.